0maroo0/diva-app
# Feature: Implement Login Screen UI KAN-3

Jira Ticket: [KAN-3](https://diva-shopping.atlassian.net/browse/KAN-3?atlOrigin=eyJpIjoiZTFlMzFhMTAyMGUyNDM5ZDhkOWU1MjIxNGU4ZGFmZjAiLCJwIjoiaiJ9)

### Description 📖:
- Implemented the UI for the Login Screen with custom colors, images, SVGs, and fonts.

### How to Test 🧪:
- Run the app and navigate to the Login Screen to verify the design and functionality.
